tests/ui/future-sizes/large-arg.rs

```diff
@@ -0,0 +1,17 @@
+// compile-flags: -Z print-type-sizes --crate-type=lib
+// edition: 2021
+// build-pass
```

Comment:
The `// ignore-pass` is necessary to skip a "check" build, which causes the UI test failure.

```suggestion
// build-pass
// ignore-pass
```

Reply:
thanks yeah I saw it earlier when I copied one of the tests over but didn’t quite understand what it’s purpose was (I did see a small explanation in the Rustc Dev Guide)

but now I get what it’s for, thanks :)